api/v1alpha1/loadbalancer_types.go

```diff
@@ -24,6 +27,15 @@ type LoadBalancer struct {
 	//
 	// +optional
 	ConsistentHash *ConsistentHash `json:"consistentHash,omitempty"`
+
+	// SlowStartWindow defines the duration of the warm up period for newly added host.
+	// If set, during slow start window, traffic sent to the newly added hosts will gradually increase in a linear manner.
+	// Currently this is only supported for RoundRobin and LeastRequest load balancers.
+	// For additional details,
+	// see https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/cluster.proto#config-cluster-v3-cluster-slowstartconfig
+	//
+	// +optional
+	SlowStartWindow *metav1.Duration `json:"slowStartWindow,omitempty"`
```

Comment:
another question regarding API - should it be `slowStart.window` to accommodate for future fields (like `aggression` ) or do we think that its not needed for the common use cases ?

Reply:
IMO, currently, for most use cases, linear growth of traffic is enough. If there are many requirements for non-linear growth later, we will expand it. 😄

Reply:
I vote to make it a struct just in case we need to expand in the future

Reply:
done